clang/tools/dpct/cmake/dpct.cmake

```diff
@@ -78,3 +78,9 @@ endmacro()
 # Always set SYCL_HAS_FP16 to true to assume SYCL device to support float16
 message("dpct.cmake: SYCL_HAS_FP16 is set true by default.")
 set(SYCL_HAS_FP16 TRUE)
+
+if(WIN32)
```

Comment:
Pls submit behavior test for DNN_LIB in SYCLomatic-test, and sycl version test in https://github.com/intel-sandbox/cmake_script_migration/tree/master/lit/target_link_libraries/ to verify DNN_LIB

Reply:
Behavior test for target_link_libs: https://github.com/oneapi-src/SYCLomatic-test/pull/577